lib/profileAgents.js

```diff
@@ -71,11 +71,11 @@ exports.create = async ({accountId, profileId, token} = {}) => {
   assert.optionalString(token, 'token');
 
   if(accountId && token) {
-    throw new TypeError(
```

Comment:
Why the change from `TypeError` to `Error`? What problem is being solved here?

Reply:
@gannan08 it didn't look correct to use TypeError here since we're not doing a type check. I'm open to suggestions or change it change back..

Reply:
Web APIs typically use `TypeError` when parameters are missing, i.e., the type of the param is `undefined`. We should stick to doing that.